addOns/pscanrules/CHANGELOG.md

```diff
@@ -38,6 +38,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - The CWE for the Cookie Poisoning scan rule was updated to a more specific one.
 - The Strict-Transport-Security Header and Big Redirect scan rules now use alert references for their different types of alerts (Issue 7100).
 
+## [54] - 2024-01-15
+### Changed
+- Update Vulnerabilities references to use https links and retire some which were out-dated (Issue 8262).
+
```

Comment:
It should be an entry with `Retrieved from Cache` under `The references for the following scan rules are now all HTTPS (Issue 8262) and in some cases updated:`.

Reply:
Ok ,
Thankyou for the help much needed

Reply:
It should be added under the existing entry though. (The version/date is automatically updated when the add-ons are released.)

Reply:
With existing entry I meant line 9, it just needs to be listed like all the other scan rules.

Reply:
Done.
Thank you for your penance and helping me 🙏